### ISSUE TITLE
Config/Backend.php, line 64: «Namespace for df_le class is not specified»

### ISSUE THREAD

**Issue by dmitry-fedyuk** [mage2pro/core#27]
I have the following code there:
```
catch (\Exception $e) {df_log($e); throw df_le($e);}
```
https://github.com/mage2pro/core/blob/2.12.20/Config/Backend.php#L64

`df_le` is not a class, it is a function:
```
function df_le(E $e) {return $e instanceof LE ? $e : new LE(__(df_ets($e)), $e);}
```
https://github.com/mage2pro/core/blob/2.12.20/Qa/lib/exception.php#L53-L58

**Comment by dmitry-fedyuk** [comment]
I do not know how to fix it, so I have reported it here: https://github.com/magento/marketplace-eqp/issues/61

**Comment by dmitry-fedyuk** [comment]
Another issue with the «Namespace for class is not specified» message https://github.com/magento/marketplace-eqp/issues/45
It is much easier to fix than mine...

**Comment by dmitry-fedyuk** [comment]
I have suppressed it in the [2.12.23](https://github.com/mage2pro/core/tree/2.12.23) version by `// @codingStandardsIgnoreLine`.

**Comment by dmitry-fedyuk** [comment]
The Magento Marketplace's automatic technical validation tool ignores: `// @codingStandardsIgnoreLine` (so I need another solution):
![01](https://user-images.githubusercontent.com/83204/30929035-ef416076-a3c5-11e7-9ad7-af598857a516.png)

**Comment by dmitry-fedyuk** [comment]
Fixed in the [2.12.27](https://github.com/mage2pro/core/tree/2.12.27) version.
